pom.xml

```diff
@@ -44,7 +44,7 @@
         <context-root>deposit</context-root>
         <skipTests>false</skipTests>
         <output.directory>target/build</output.directory>
-        <easy-licenses.version>1.0.1</easy-licenses.version>
+        <easy.licenses.version>1.0.2</easy.licenses.version>
```

Comment:
Why change `easy-licenses` to `easy.licenses`?  The module is called `easy-licenses`, so the previous name for this property make more sense to me.

Reply:
i followed the convention used in the other modules that import this. this one was the odd one out

Reply:
OK, fair enough.